_SQLServer/TableLoadUtilities.sql

```diff
@@ -0,0 +1,47 @@
+--1.0
+--Utility (Used Utility 2.0 from TableLoad_Scratch file)
+
+
+--Difficult Terrain
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor)
+VALUES ('Difficult Terrain',
+'http://htkb.info/SDE/1.0/Cards/Utility/Card_Utility_Terrain_Difficult_Terrain_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Utility/Card_Utility_Terrain_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Utility/Card_Utility_Terrain_Difficult_Terrain.xhtml', 
+'Utility', '1.0', '1.0 Core Set', 'Default', '');
+insert into Utilities (CardIndex, UtilityDescription) VALUES (43, ''); --? Not sure what to put here
```

Comment:
UtilityDescription should be: 'Each square of difficult terrain a model enters costs two points of Speed.  If a model does not have enough Speed remaining to move into a square with difficult terrain it may not do so.'